nltk/translate/rouge_score.py

```diff
@@ -0,0 +1,207 @@
+'''ROUGE score implementation
+Link to paper:
+http://www.aclweb.org/anthology/W04-1013
+'''
+
+from nltk.util import ngrams
+from nltk.tokenize import TreebankWordTokenizer, PunktSentenceTokenizer
+from nltk import skipgrams
+from nltk.translate.util import jacknifing, lcs
+import numpy as np
+
+
+tokenizer = TreebankWordTokenizer()
+sentence_tokenizer = PunktSentenceTokenizer()
+
```

Comment:
The `rogue_*()` should follow the assumption that the strings are already tokenized and the implementation in NLTK is taking the tokenized string to produce the score.

Reply:
Done